polar2grid/readers/clavrx.py

```diff
@@ -64,102 +64,87 @@
-        'cloud_type'
-        'cld_temp_acha',
-        'cld_height_acha',
-        'cloud_phase',
-        'cld_opd_dcomp',
-        'cld_opd_nlcomp',
-        'cld_reff_dcomp',
-        'cld_reff_nlcomp',
-        'cld_emiss_acha',
-        'refl_lunar_dnb_nom',
-        'rain_rate',
-    ]
-
-    @property
-    def default_products(self):
-        return set(self.DEFAULT_DATASETS) & set(self.available_product_names)
-
-    @property
-    def available_product_names(self):
-        available = set(self.scene.available_dataset_names(reader_name=self.reader, composites=True))
-        return sorted(available & set(self.all_product_names))
-
-    @property
-    def all_product_names(self):
-        return ADVERTISED_DATASETS & set(self.scene.all_dataset_names(reader_name=self.reader))
-
-    def filter(self, scene):
-        self.filter_daynight_datasets(scene)
-
-    def filter_daynight_datasets(self, scene):
-        """Some products are only available at daytime or nighttime"""
-        for k in DAY_ONLY + NIGHT_ONLY:
-            if k in scene and scene[k].isnull().all():
-                LOG.info("Removing dataset '{}' because it is completely empty".format(k))
-                del scene[k]
-
-
-def add_frontend_argument_groups(parser):
-    """Add command line arguments to an existing parser.
-
-    :returns: list of group titles added
-    """
-    from polar2grid.core.script_utils import ExtendAction
-    # Set defaults for other components that may be used in polar2grid processing
-    parser.set_defaults(fornav_D=40, fornav_d=1)
-
-    # Use the append_const action to handle adding products to the list
-    group_title = "Frontend Initialization"
-    group = parser.add_argument_group(title=group_title, description="swath extraction initialization options")
-    group.add_argument("--list-products", dest="list_products", action="store_true",
-                       help="List available frontend products and exit")
-    group_title = "Frontend Swath Extraction"
-    group = parser.add_argument_group(title=group_title, description="swath extraction options")
-    group.add_argument("-p", "--products", dest="products", nargs="+", default=None, action=ExtendAction,
-                       help="Specify frontend products to process")
-    return ["Frontend Initialization", "Frontend Swath Extraction"]
-
-if __name__ == "__main__":
-    sys.exit(main(description="Extract CLAVR-X swath data into binary files",
-                  add_argument_groups=add_frontend_argument_groups))
+    ADVERTISED_DATASETS = set(
+        [
+            "cloud_type",
+            "cld_temp_acha",
+            "cld_height_acha",
+            "cloud_phase",
+            "cld_opd_dcomp",
+            "cld_opd_nlcomp",
+            "cld_reff_dcomp",
+            "cld_reff_nlcomp",
+            "cld_emiss_acha",
+            "refl_lunar_dnb_nom",
+            "rain_rate",
+        ]
+    )
+
+DEFAULT_DATASETS = [
+    "cloud_type" "cld_temp_acha",
```

Comment:
Looks like this was a typo in the original list. This should have a comma between it it seems.